xwrf/postprocess.py

```diff
@@ -24,6 +24,14 @@ def _decode_times(ds: xr.Dataset) -> xr.Dataset:
         )
     ds = ds.assign_coords({'Time': _time})
     ds.Time.attrs = {'long_name': 'Time', 'standard_name': 'time'}
+    # make XTIME be consistent with its description
+    if 'XTIME' in ds.variables and ds.XTIME.dtype == 'datetime64[ns]':
```

Comment:
```suggestion
    if 'XTIME' in ds.variables and np.issubdtype(ds.XTIME.dtype, np.datetime64):
```

Reply:
Interestingly would it help to pass `decode_timedelta=True` in open_dataset? 

https://github.com/pydata/xarray/pull/4071/files

In general your input on [this issue](https://github.com/pydata/xarray/issues/1621) would be  very valuable.